pytest_fixtures/core/broker.py

```diff
@@ -173,3 +173,91 @@ def module_capsule_configured(module_capsule_host, module_target_sat):
     """Configure the capsule instance with the satellite from settings.server.hostname"""
     module_capsule_host.capsule_setup(sat_host=module_target_sat)
     yield module_capsule_host
+
+
+@pytest.fixture(scope='module')
+def module_capsule_configured_mqtt(module_capsule_configured):
```

Comment:
these mqtt fixtures aren't intended for 6.11.z, so I think you should add only `module_discovery_sat`

Reply:
Yeah I don't know why I did that haha